app_core/migrations/versions/20260524_add_dashboard_branding_to_application_settings.py

```
@@ -0,0 +1,63 @@
+revision = "20260524_add_dashboard_branding_to_application_settings"
+down_revision = "20260522_add_system_metric_samples"
+branch_labels = None
+depends_on = None
+
+
+def _column_exists(table_name: str, column_name: str) -> bool:
+    conn = op.get_bind()
+    inspector = inspect(conn)
+    try:
+        cols = {c["name"] for c in inspector.get_columns(table_name)}
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)